nemo/collections/nlp/models/token_classification/ner_model.py

```diff
@@ -85,9 +85,9 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
 
         if cfg.class_balancing == 'weighted_loss':
             # You may need to increase the number of epochs for convergence when using weighted_loss
-            self.loss = CrossEntropyLoss(weight=self.data_desc.class_weights)
+            self.loss = CrossEntropyLoss(logits_ndim=3, weight=self.data_desc.class_weights)
```

Comment:
@ekmb is this change OK?

Reply:
I asked her on slack and she said its corect. Same for below.